### PR TITLE
Fix for missing ResourceRecord field for Domains and fix warnings

### PR DESCRIPTION
**Pull request by pushpakpop**

### CODE REVIEW

File: inc/admin/namespace.php [hunk 1]
```diff
@@ -191,9 +191,9 @@ function admin_page() {
 				<tbody>
 					<?php foreach ( $certificate['DomainValidationOptions'] as $domain ) : ?>
 						<tr>
-							<td><?php echo esc_html( $domain['ResourceRecord']['Name'] ) ?></td>
-							<td><?php echo esc_html( $domain['ResourceRecord']['Type'] ) ?></td>
-							<td><?php echo esc_html( $domain['ResourceRecord']['Value'] ) ?></td>
+							<td><?php echo esc_html( isset( $domain['ResourceRecord']['Name'] ) ? $domain['ResourceRecord']['Name'] : '' ) ?></td>
```
Comment (rmccue) [inline inc/admin/namespace.php]: `$domain['ResourceRecord']['Name'] ?? ''` would be more succinct and clearer here I think.

File: inc/namespace.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ function create_certificate( array $domains ) : array {
 
 	$arn = $result['CertificateArn'];
 
-	sleep( 3 ); // What a great hack.
+	sleep( 10 ); // What a great hack. // Allows enough time for AWS to generate ResourceRecord for provided domains before we call 'describeCertificate' for the certificate at hand.
```
Comment (rmccue) [inline inc/namespace.php]: Needs a new line for a new comment, or just remove the old (useless) text.

**Review by rmccue — DISMISSED** [review]

File: inc/admin/namespace.php (hunk 1 above)
Comment (pushpakpop) [reply inc/admin/namespace.php]: Didn't used it just for old version php compatibility. @rmccue Do let me know if I should still update it.

Kindest Regards,

File: inc/namespace.php (hunk 2 above)
Comment (pushpakpop) [reply inc/namespace.php]: Updated comment and placement.

Kindest regards,

File: inc/admin/namespace.php (hunk 1 above)
Comment (pushpakpop) [reply inc/admin/namespace.php]: Updated :)

**Review by nprasath002 — APPROVED** [review]
LGTM

**Pull request merged**